src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Utils/Extensions/DataSetVersionExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Utils.Extensions;
+
+public static class DataSetVersionExtensions
+{
+    public static DataSetVersionType VersionType(this DataSetVersion dataSetVersion)
+    {
+        if (dataSetVersion.VersionMinor == 0)
+        {
+            return DataSetVersionType.Major;
+        }
+
+        return DataSetVersionType.Minor;
+    }
+}
```

Comment:
Wouldn't do this as an extension. It makes more sense being on the `DataSetVersion` model itself as a getter method so we can just call it like:

```cs
dataSetVersion.VersionType;
```

The `VersionType` may be useful in other situations later on, so it'd be better for it to be part of the domain model.

Reply:
I was actually going to bring this up with you to come to an agreement.

The problem with this is that `DataSetVersionType` is in `GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels` as it's part of the view model. So the `Models` project would need a circular reference back to the `Api` project in this case. Which is why I added it as an extension rather than on the model itself.

An alternative would be to put `DataSetVersionType` in `Models` which means I could have this method on the model without adding that circular dependency. So if we're happy with `DataSetVersionType` being a **domain model** rather than a **view model** then this is the route to go down.

What are your thoughts?

Reply:
agreed to move to `Models` :)